src/components/v5/common/ActionSidebar/partials/UserSelect/UserSelect.tsx

```diff
@@ -32,15 +35,21 @@ const UserSelect: FC<UserSelectProps> = ({ name }) => {
     },
   ] = useToggle();
   const { user: userByAddress } = useUserByAddress(field.value);
-  const userDisplayName = userByAddress?.profile?.displayName;
+  const { user: userByName } = useUserByName(field.value);
```

Comment:
Are you sure we need `useUserByName` here? I believe we keep only the wallet address in `field.value`